[rocalution] cannot find rocrand

```
==> Starting build()...
-- Setting build type to 'Release' as none was specified.
-- The CXX compiler identification is Clang 12.0.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/rocm/hip/bin/hipcc - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /usr/bin/git (found version "2.29.2") 
-- Found OpenMP_CXX: -fopenmp=libomp  
-- Found OpenMP: TRUE   
-- Could NOT find MPI_CXX (missing: MPI_CXX_LIB_NAMES MPI_CXX_HEADER_DIR MPI_CXX_WORKS) 
-- Could NOT find MPI (missing: MPI_CXX_FOUND) 
    Reason given by package: MPI component 'C' was requested, but language C is not enabled.  MPI component 'Fortran' was requested, but language Fortran is not enabled.  

-- MPI not found. Compiling WITHOUT MPI support.
-- Found HIP: /opt/rocm/hip (found version "3.10.") 
-- Performing Test HIP_CLANG_SUPPORTS_PARALLEL_JOBS
-- Performing Test HIP_CLANG_SUPPORTS_PARALLEL_JOBS - Success
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
CMake Error at cmake/Dependencies.cmake:87 (find_package):
  By not providing "Findrocrand.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "rocrand", but
  CMake did not find one.

  Could not find a package configuration file provided by "rocrand" with any
  of the following names:

    rocrandConfig.cmake
    rocrand-config.cmake

  Add the installation prefix of "rocrand" to CMAKE_PREFIX_PATH or set
  "rocrand_DIR" to a directory containing one of the above files.  If
  "rocrand" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:74 (include)


-- Configuring incomplete, errors occurred!
See also "/build/rocalution/src/build/CMakeFiles/CMakeOutput.log".
See also "/build/rocalution/src/build/CMakeFiles/CMakeError.log".
==> ERROR: A failure occurred in build().
```

--- a/rocalution/.SRCINFO
+++ b/rocalution/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = rocalution
 	pkgdesc = Next generation library for iterative sparse solvers for ROCm platform
 	pkgver = 3.10.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://rocalution.readthedocs.io/en/master
 	arch = x86_64
 	license = MIT
@@ -11,7 +11,7 @@ pkgbase = rocalution
 	depends = rocsparse
 	depends = rocblas
 	depends = rocprim
-	depends = rocminfo
+	depends = rocrand
 	depends = openmp
 	source = rocalution-3.10.0.tar.gz::https://github.com/ROCmSoftwarePlatform/rocALUTION/archive/rocm-3.10.0.tar.gz
 	sha256sums = c24cb9d1a8a1a3118040b8b16dec7c06268bcf157424d3378256cc9eb93f1b58

--- a/rocalution/PKGBUILD
+++ b/rocalution/PKGBUILD
@@ -1,12 +1,12 @@
 # Maintainer: Markus Näther <naetherm@informatik.uni-freiburg.de>
 pkgname=rocalution
 pkgver=3.10.0
-pkgrel=1
+pkgrel=2
 pkgdesc='Next generation library for iterative sparse solvers for ROCm platform'
 arch=('x86_64')
 url='https://rocalution.readthedocs.io/en/master'
 license=('MIT')
-depends=('hip-rocclr' 'rocsparse' 'rocblas' 'rocprim' 'rocminfo' 'openmp')
+depends=('hip-rocclr' 'rocsparse' 'rocblas' 'rocprim' 'rocrand' 'openmp')
 makedepends=('cmake' 'git')
 _git='https://github.com/ROCmSoftwarePlatform/rocALUTION'
 source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz")

